tests/unit/test_sql_execution_internal.py

```
@@ -377,6 +377,228 @@ def test_create_sql_ssh_uri_no_ssh():
         assert url is None
 
 
+def _make_sql_alchemy_dict(integration_id="integration_a", url=None, params=None):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_make_sql_alchemy_dict`

(ANN202)